lib/dialects/mssql/query-generator.js

```diff
@@ -782,101 +805,38 @@ class MSSQLQueryGenerator extends AbstractQueryGenerator {
     return 'ROLLBACK TRANSACTION;';
   }
 
-  selectFromTableFragment(options, model, attributes, tables, mainTableAs, where) {
-    let topFragment = '';
-    let mainFragment = `SELECT ${attributes.join(', ')} FROM ${tables}`;
-
-    // Handle SQL Server 2008 with TOP instead of LIMIT
-    if (semver.valid(this.sequelize.options.databaseVersion) && semver.lt(this.sequelize.options.databaseVersion, '11.0.0')) {
```

Comment:
Removal of MSSQL <11.0.0 support should probably be well documented in the affected methods.

Reply:
Sequelize will throw an error before connecting to MSSQL < 11. The error is thrown by `versionSupport` in `lib/dialects/mssql/connection-manager.js`.